rllib/models/tests/test_distributions.py

```diff
@@ -23,3 +23,8 @@ def test_categorical(self):
             counts[sample] += 1.0
         probs = np.exp(z) / np.sum(np.exp(z))
         self.assertTrue(np.sum(np.abs(probs - counts / num_samples)) <= 0.01)
+
+
+if __name__ == "__main__":
+    import unittest
+    unittest.main(verbosity=1)
```

Comment:
Prefer using pytest, since it suppresses non-error logs:

```
if __name__ == "__main__":
    import pytest
    import sys
    sys.exit(pytest.main(["-v", __file__]))
```